### DISCUSSION TITLE
feedback package permissions needed

### DISCUSSION THREAD

**Discussion by DktPhl2019** [ueman/feedback#255] [Q&A]
Hello!
If I use this package, what permissions do I need to include in the xml file?
I need a user to: 1) save image to local phone after taking a screenshot 2) send email to a developer.
Could you please specify. 
Thank You

**Comment by ueman** [comment]
This depends on your implementation of how the feedback is send and the target platform, but answering questions in regards to that is out of scope for this package.

The feedback package itself doesn't need any permission, because it doesn't do anything that could require a permission regardless of the target platform.
